docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -1,6 +1,21 @@
 <uxd-page-header header="Getting Started"></uxd-page-header>
 <uxd-text-page-layout>
-    <h3>Installation</h3>
+
+    <h3 id="add-to-an-existing-project">Add to an Existing Project</h3>
+
+    <p>
+        To add UX Aspects (AngularJS) to your Angular CLI or Nx workspace, run the following command in the project root:
+    </p>
+
+    <pre>
+ng add @ux-aspects/ux-aspects-ng1 --registry https://svsartifactory.swinfra.net/artifactory/api/npm/company-central-npm/</pre>
```

Comment:
We actually don't need `--registry` in this case since we are only deploying open source components.